tests/server/test_bundle_service.py

```diff
@@ -0,0 +1,210 @@
+import copy
+
+from fastapi.testclient import TestClient
+from requests import Response
+
+from bundle_service.main import app
+
+client = TestClient(app)
+
+HEADERS = {"Authorization": "foo"}
```

Comment:
Guessing this would normally carry an authorization token from fence?

Reply:
Correct.  It would be inserted by nginx/revproxy